helm_deploy/hmpps-registers/values.yaml

```diff
@@ -1,5 +1,47 @@
 ---
 # Values here are the same across all environments
 
+generic-service:
+  nameOverride: hmpps-registers
+  replicaCount: 2
```

Comment:
I know the `values-prod` was set to `2` but for all our production services it should be `4`.

Reply:
note that you'll need to then override for `dev` to be 2

Reply:
ah ok - I didn't realise that.